src/stories/HsitorySelect/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import {Button, ButtonProps, HistorySelectionDialogTitle, SelectionModal} from '../../components';
+
+export interface HistorySelectionProps extends ButtonProps{}
+
+export default function HistorySelection() {
+    const [open, setOpen] = useState(false);
+
+    const handleClickOpen = () => {
+        setOpen(true);
+    };
+    const handleClose = () => {
+        setOpen(false);
+    };
+    return (
+        <>
+            <Button variant="contained" className="History-selection" onClick={handleClickOpen}>

```

Comment:
if we go with BEM, then it's better to follow its notation, for instance use className="History-selection__open"